src/utils/config/ethereum.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from '../../types';
 
 const SUBGRAPH_URL =
-    'https://api.thegraph.com/subgraphs/name/salazarguille/yearn-vaults-v2-subgraph-mainnet';
```

Comment:
@salazarguille 
Is it okay to update the subgraph URL we are using?
would you please confirm?